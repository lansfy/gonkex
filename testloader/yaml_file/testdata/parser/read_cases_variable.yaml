- name: loader MUST read tests with variables
  method: POST
  path: /some/path
  request: requestbody
  response:
    200: someresponsebody
  cases:
    - variables:
        foo1: bar1
        foo2: bar2
    - variables:
        foo1: bar21
        foo3: bar22
