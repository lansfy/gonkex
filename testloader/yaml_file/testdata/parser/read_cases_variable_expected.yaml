- GetName: "loader MUST read tests with variables #1"
  GetMethod: POST
  Path: /some/path
  GetRequest: requestbody
  GetResponses:
    200: "someresponsebody"
  GetCombinedVariables:
    foo1: bar1
    foo2: bar2
  GetFileName: testdata/parser/read_cases_variable.yaml
  OneOfCase: true
  FirstTestInFile: true

- GetName: "loader MUST read tests with variables #2"
  GetMethod: POST
  Path: /some/path
  GetRequest: requestbody
  GetResponses:
    200: "someresponsebody"
  GetCombinedVariables:
    foo1: bar21
    foo2: bar2
    foo3: bar22
  GetFileName: testdata/parser/read_cases_variable.yaml
  OneOfCase: true
  LastTestInFile: true
